lib/src/st_tx_video_session.c

```diff
@@ -2092,6 +2091,9 @@ static int tv_attach(struct st_main_impl* impl, struct st_tx_video_sessions_mgr*
     return ret;
   }
 
+  s->st20_linesize = ops->width * s->st20_pg.size / s->st20_pg.coverage;
+  if (ops->linesize > s->st20_linesize) s->st20_linesize = ops->linesize;
```

Comment:
Pls return error if ops->linesize is invalid(<ops->width * s->st20_pg.size / s->st20_pg.coverage)